spring-kafka/src/main/java/org/springframework/kafka/listener/ConsumerProperties.java

```diff
@@ -501,6 +508,23 @@ public void setCheckDeserExWhenValueNull(boolean checkDeserExWhenValueNull) {
 		this.checkDeserExWhenValueNull = checkDeserExWhenValueNull;
 	}
 
+	/**
+	 * Return the container's group protocol.
+	 * @return Container group protocol set via the group.protocol Consumer property
+	 */
+	public String getGroupProtocol() {
+		return this.groupProtocol;
+	}
+
+	/**
+	 * Set the container group protocol.
+	 *
```

Comment:
No blank lines in method Javadocs.
The `@since` is supposed to be there on the public API.
And why is this property is not set on the target `KafkaConsumer` in this PR?